src/test/mochitest/browser_dbg-search-sources.js

```diff
@@ -13,7 +13,6 @@ add_task(async function() {
 
   pressKey(dbg, "sourceSearch");
   await waitForElement(dbg, "input");
-  findElementWithSelector(dbg, "input").focus();
```

Comment:
We don't need to manually focus our elements they do that themselves. Leaving these in caused failures.

Reply:
nice